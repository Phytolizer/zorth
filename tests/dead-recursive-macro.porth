include "std.porth"

macro dead
  dead
end

"The above macro is recursive, but it is OK because it is never expanded.\n" stdout write
