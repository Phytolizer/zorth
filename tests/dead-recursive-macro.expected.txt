:i returncode 0
:b stdin 0

:b stdout 73
The above macro is recursive, but it is OK because it is never expanded.

:b stderr 0

